client/src/templates/Challenges/components/challenge-description.css

```diff
@@ -2,13 +2,28 @@
   font-size: 16px;
 }
 
-.challenge-instructions blockquote {
+ .challenge-instructions blockquote, .challenge-instructions pre, .challenge-instructions pre[class*="language-"] {
```

Comment:
@ojeytonwilliams Why do we need the last selector?

Reply:
Is `.challenge-instructions pre` not enough?

Reply:
@ValeraS We tried that originally, but it did not work.

Reply:
Prism has these selectors: `code[class*="language-"], pre[class*="language-"]`
`.challenge-instructions pre` selector weights more.

Triple backticks with lang become `<pre><code class="language-lang">...`

What construction becomes `<pre class ="language-...">`?

Reply:
@ojeytonwilliams As a test, I removed just `.challenge-instructions pre[class*="language-"] ` as a selector and left all other CSS the same in the `challenge-descriptions.css` file.  It seems to be fine.  Can you validate also?  I am not sure why I thought it did not work originally, but maybe it had to do with other CSS we had not added at that point.

Reply:
> What construction becomes `<pre class ="language-...">`?

@ValeraS  Good point - none of the constructions we're trying to support.  You would have to put pre tags in manually, rather than than via backticks, so we don't need this.

@RandellDawson Exactly.  We needed some more css, just not this bit.